src/reconcile/rec_visibility.c

```diff
@@ -44,6 +44,7 @@ __rec_update_save(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, WT_
     supd->ins = ins;
     supd->rip = rip;
     supd->onpage_upd = onpage_upd;
+    supd->tombstone = tombstone;
```

Comment:
Now we have the tombstone, I think it's better to save the tombstone that deletes the key in this pointer. We are using selected_upd to delete the key still, which is actually odd.

Reply:
Decided not to go with this for the same reason as on another comment, resolving this conversation.